shared/output/cloud/vision/lib/google/cloud/vision/v1/image_annotator_client.rb

```diff
@@ -0,0 +1,288 @@
+# frozen_string_literal: true
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "json"
+require "pathname"
+
+require "googleauth"
+require "google/gax"
+require "google/gax/operation"
+require "google/longrunning/operations_client"
+
+require "google/cloud/vision/v1/image_annotator_pb"
+
+module Google
+  module Cloud
+    module Vision
+      module V1
+        module ImageAnnotator
+          class Credentials < Google::Auth::Credentials
+            SCOPE = [
+              "https://www.googleapis.com/auth/cloud-platform"
+            ].freeze
+            PATH_ENV_VARS = %w[IMAGEANNOTATOR_CREDENTIALS
```

Comment:
`IMAGEANNOTATOR_CREDENTIALS` - This is incorrect, but will be addressed later by #59?

Reply:
Correct.